### PR TITLE
Adds waiver option to gb to ni calculation result

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-576

### What?

I have added/removed/altered:

- [x] Adds waiver default option for GB to NI duty calculations
- [x] Rename config/locales/measure_types.en.yml -> config/locales/footnotes.en.yml

### Why?

I am doing this because:

- This is required to indicate that under certain circumstances the duties into NI can be waived

### ROW -> UK duty calculation

Screenshot from 2021-04-21 10-17-56

### GB -> NI duty calculation

Screenshot from 2021-04-21 10-20-13

### CODE REVIEW

File: app/models/duty_calculator.rb [hunk 1]
```diff
@@ -57,4 +57,18 @@ def additional_duty_rows
         acc << option_klass.new(measure, user_session, []).option
       end
   end
+
+  def default_options
+    return [waiver_option] if user_session.gb_to_ni_route?
+
+    []
+  end
```
Comment (tavy88) [inline app/models/duty_calculator.rb:65]: oh, interesting so it's only on GB to NI route.

File: app/models/duty_calculator.rb [hunk 2]
```diff
@@ -57,4 +57,18 @@ def additional_duty_rows
...
+
+  def waiver_option
+    {}.tap do |option|
+      option[:key] = DutyOptions::Waiver.id
+      option[:evaluation] = DutyOptions::Waiver.new(nil, user_session, []).option
+      option[:priority] = DutyOptions::Waiver::PRIORITY
+    end
```
Comment (tavy88) [inline app/models/duty_calculator.rb:72]: Why not just: 

```ruby
{
   key: DutyOptions::Waiver.id,
  evaluation: DutyOptions::Waiver.new(nil, user_session, []).option,
 ...
}
```

Any reason why we need to use tap?

**Review by tavy88 — APPROVED** [review]

File: app/models/duty_calculator.rb (hunk 1 above)
Comment (willfish) [reply app/models/duty_calculator.rb:65]: Yarp

File: app/models/duty_calculator.rb (hunk 2 above)
Comment (willfish) [reply app/models/duty_calculator.rb:72]: So no good reason beyond I always wanted to return the option.

**Pull request merged**